integration-cli/docker_cli_volume_test.go

```diff
@@ -87,23 +131,23 @@ func (s *DockerSuite) TestVolumeCliLsFilterDangling(c *check.C) {
 	out, _ := dockerCmd(c, "volume", "ls")
 
 	// No filter, all volumes should show
-	c.Assert(out, checker.Contains, "testnotinuse1\n", check.Commentf("expected volume 'testnotinuse1' in output"))
```

Comment:
@cpuguy83 I can implement @calavera suggestion but I think it is needed to see container associations from the volume commands to make processing simpler. I agree with your point on this being slow and all, but I think maintaining such list and syncing would be major change compared to building the list dynamically. I think at the moment either we extend 'ls' or 'inspect' to help this use case.